package.json

```diff
@@ -10,7 +10,11 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@mui/icons-material": "^5.15.20",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@fortawesome/fontawesome-free": "^6.5.2",
+    "@mui/icons-material": "^5.15.21",
+    "@mui/material": "^5.15.21",
```

Comment:
We need to remove MUI related packages and use an icon package like lucide-react as we are only using mui for icons.